Dockerfile

```diff
@@ -53,3 +52,35 @@ COPY extend-environment.sh /etc/profile.d/extend-environment.sh
 RUN pip install --no-cache-dir -r requirements/local.txt
 
 CMD ["python", "./cfgov/manage.py", "runserver", "0.0.0.0:8000"]
+
+FROM cfgov-develop as cfgov-build
+
+# add node and yarn repos
+RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
+RUN curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo
+
+RUN yum install -y nodejs yarn  && \
+    yum clean all && rm -rf /var/cache/yum
+
+WORKDIR /src/cfgov-refresh
+
+COPY scripts scripts
+COPY frontend.sh .
+COPY package.json .
+COPY yarn.lock .
+COPY babel.config.js .
+COPY gulpfile.js .
+COPY gulp ./gulp
+COPY config ./config
+COPY jest.config.js .
+COPY cfgov/ ./cfgov/
+COPY static.in ./static.in
+
+ENV DJANGO_SETTINGS_MODULE=cfgov.settings.production
+ENV ALLOWED_HOSTS='["*"]'
+
+RUN sh ./frontend.sh production
+
+RUN DJANGO_STATIC_ROOT=/var/www/html/static cfgov/manage.py collectstatic
```

Comment:
Why is `DJANGO_STATIC_ROOT` not an `ENV`?

Reply:
Because it's only used once?

Reply:
Ah.  OK.  I thought it was used when the app was running too.

Reply:
that's in theory possible-- so maybe I will set it with ENV. In general, I think this image is just for producing assets that will get copied to the production image

Reply:
Oh right, the production image won't actually inherit from this one.  You _could_ move it into `cfgov-base`, then it'd be available in both.  Might be nice so you don't have to set it twice, or remember to change it in two places.  Or we could wait to figure that out when we do the full Apache work since I'm guessing we'll have other `ENV`s that this could build off of.

Reply:
so we set it in base, but override it in deveop?

Reply:
That doesn't really work either, though, since `-build` inherits from `-develop`.  You'd just end up re-resetting it.

I say just keep it as-is for this PR, and we'll probably have a better idea of how all the stages will fit together when we do that Apache work.

Reply:
OK